src/main/java/dateBases/entitys/Developer.java

```diff
@@ -0,0 +1,27 @@
+package dateBases.entitys;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Developer {
+    private Integer id;
+    private String surName;
+    private String firstName;
+    private Integer age;
+    private Boolean isMale;
+    private Integer salary;
+
+    @Override
+    public String toString() {
```

Comment:
In lombok you can fina annotation  @ToString. It would help to make this class smaller.
https://projectlombok.org/features/ToString